cli/js/workers_test.ts

```diff
@@ -76,7 +76,7 @@ test(async function workerThrowsWhenExecuting(): Promise<void> {
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
   throwingWorker.onerror = (e: any): void => {
     e.preventDefault();
-    assertEquals(e.message, "Uncaught Error: Thrown error");
+    assert(/Uncaught Error: Thrown error/.test(e.message));
```

Comment:
Is there some additional info attached now?

Reply:
Yeah it's a colorful error now:

![image](https://user-images.githubusercontent.com/218257/75500459-deb0bf80-5981-11ea-9c7f-5e87ba70dba4.png)

Granted, "error: Uncaught Error: ..." is redundant and rather ugly, but we output that any many places, so I think this is not indicative of a problem with this particular PR.